platform/windows/os_windows.cpp

```diff
@@ -824,6 +824,33 @@ Error OS_Windows::shell_open(String p_uri) {
 	}
 }
 
+Error OS_Windows::shell_show_in_explorer(String p_fpath) {
+	if (!p_fpath.begins_with("file://")) {
+		p_fpath = String("file://") + p_fpath;
+	}
+	INT_PTR ret = (INT_PTR)ShellExecuteW(nullptr, nullptr, LPCWSTR(String("explorer.exe").utf16().get_data()), LPCWSTR((String("/select,") + p_fpath).utf16().get_data()), nullptr, SW_SHOWNORMAL);
```

Comment:
`String("/select,") + p_fpath`, is it working if path contains spaces? If not, it might need escaping.

Reply:
Ok,  I use `String("file://") + p_fpath.strip_escapes().strip_edges()` to replace  it. I am not sure it is necessary that using `strip_escapes()` at here.